src/main/steps/applicant2/check-your-joint-application/content.ts

```diff
@@ -111,11 +56,26 @@ export const form: FormContent = {
   },
 };
 
+const getApplicant1PartnerContent = (content: CommonContent): string => {
```

Comment:
doesnt the common content file accomdate for this?

Reply:
@velimd this is the reverse of what the common content file does because this is where we get what the currently logged in user is referred to by their partner